ebi-deployment/README.md

```diff
@@ -0,0 +1,51 @@
+# EBI-deployment
+
+Helm chart to deploy EGA-DATA-API to any Kubernetes cluster.
+
+## Table of Contents
+
+- [Prerequisites](#Prerequisites)
+- [Installing the Chart](#Installing-the-Chart)
+- [Uninstalling the Chart](#Uninstalling-the-Chart)
+
+## Prerequisites
+
+Before deploying the Helm charts `Helm` should be installed. Please follow [these ](https://docs.helm.sh/using_helm#install-helm) instructions to install `Helm`.
+
+Then you should also have a running Kubernetes cluster and configured correctly. You may follow [these](https://github.com/NBISweden/LocalEGA-deploy-SUNET) instructions to create a Kubernetes cluster using the tool KubeNow. It is recommended to create a node with preferably 16GB of RAM so that this Helm Chart can be deployed smoothly.
+
+Please run
+
+    helm ls
+
+to check if you Kubernetes cluster is correctly configured. If so, you should not see any error message from the above command.
+
+Finally, you should also label your node as `group1` by 
+
+    kubectl label nodes <node-name> group=group1
+
+You can get the `<node-name>` by the following command
+
+    kubectl get nodes
+
+## Installing the Chart
+
+    bash deploy.sh
```

Comment:
This is a rather bad way of doing the deployment, the shellscript servers no real purpose.
It will also fail if the first command takes to log to return.

Remove the shellscript an write the commands in the readme instead.

Reply:
Thanks for pointing out this. I've changed accordingly.